policy/modules/services/lowmemorymonitor.te

```diff
@@ -0,0 +1,36 @@
+policy_module(lowmemorymonitor)
+
+# daemon to monitor low memory and control kernel OOM
+
+########################################
+#
+# Declarations
+#
+
+type low_mem_mon_t;
+type low_mem_mon_exec_t;
+init_daemon_domain(low_mem_mon_t, low_mem_mon_exec_t)
+
+########################################
+#
+# Local policy
+#
+
+allow low_mem_mon_t self:capability ipc_lock;
+allow low_mem_mon_t self:unix_dgram_socket { create write };
```

Comment:
Can this perm set upgrade to `create_socket_perms`? It seems lacking without a `sendto`.

Reply:
ok